cdap-examples/Purchase/src/test/java/co/cask/cdap/examples/purchase/PurchaseAppTest.java

```diff
@@ -62,7 +63,8 @@ public void test() throws TimeoutException, InterruptedException, IOException {
 
     try {
       // Wait for the last Flowlet processing 5 events, or at most 5 seconds
-      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics("PurchaseHistory", "PurchaseFlow", "collector");
+      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics(Constants.DEFAULT_NAMESPACE, "PurchaseHistory",
```

Comment:
Agreed, I'll hardcode ``DEFAULT_NAMESPACE`` in ``RuntimeStats`` right now. We can figure out what to do when we want to expose namespaces to Test Framework later.